documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc

```diff
@@ -329,11 +329,11 @@ NOQuuuWuFMMMCCCCPPPPTTTTTTTTRRRR
 
 ^1^ Information on xref:raspberry-pi.adoc#otp-register-and-bit-definitions[programming the OTP bits].
 
-^2^ Warranty bit is never set on Pi4.
+^2^ Warranty bit is never set on Pi 4.
 
-=== New-style revision Codes In Use
+=== New-style Revision Codes in Use
 
-This list is not exhaustive, there may be codes in use in the field that are not in this table.
+This list is not exhaustive: there may be codes in use in the field that are not in this table.
```

Comment:
Should be semi-colon?

Reply:
And perhaps "in the wild" would be less confusing than "in the field", given that this page is discussing how to decode the bit-fields from within the revision code? :upside_down_face:

Reply:
> Should be semi-colon?

My reading of http://www.sussex.ac.uk/informatics/punctuation/colonandsemi/colon says a colon is correct here, in that the bit after the colon is "an explanation or elaboration of what precedes it".

Reply:
> And perhaps "in the wild" would be less confusing than "in the field", given that this page is discussing how to decode the bit-fields from within the revision code? 🙃

I've removed `in the field` altogether - I don't think it's really necessary.